sktime/registry/_tags.py

```diff
@@ -225,12 +243,6 @@
         "bool",
         "can the classifier set n_jobs to use multiple threads?",
     ),
-    (
-        "coerce-X-to-pandas",
-        ["classifier", "transformer"],
-        "bool",
-        "should X be coerced to a nested pandas DataFrame?",
-    ),
```

Comment:
Doesn't this and the numpy version still exist for other modules? Don't think they should be removed, if so.

Reply:
I'm pretty sure it has been removed from other modules, and tags test should verify this, but I'll double check

Reply:
Is the registry actually used elsewhere? I thought it was just to inform people about the types of tags.

Reply:
I think its part of the tests to check estimators have valid tags, in registry/tests/test_lookup